setup.py

```diff
@@ -49,14 +47,11 @@
     description=u"Planet SDK for Python",
     long_description=Path("README.md").read_text("utf-8"),
     classifiers=[
-        'Development Status :: 2 - Pre-Alpha',
-        'Environment :: Console',
+        'Development Status :: 2 - Pre-Alpha', 'Environment :: Console',
         'Intended Audience :: Developers',
         'License :: OSI Approved :: Apache Software License',
-        'Operating System :: OS Independent',
-        'Programming Language :: Python',
-        'Topic :: Scientific/Engineering',
-        'Topic :: Software Development',
+        'Operating System :: OS Independent', 'Programming Language :: Python',
+        'Topic :: Scientific/Engineering', 'Topic :: Software Development',
```

Comment:
This is the section that prompted the use of the modifier. This does't seem very readable.